Frontend/ui-common/src/Engineering/Helpers/Utils.js

```diff
@@ -10,6 +10,11 @@ export const saveToLocalStoreImpl = function(key) {
   };
 }
 
+export const compareDate = function (date1, date2) {
+  console.log("comparing : ", date1, date2);
```

Comment:
is this log required?

Reply:
We have this function already

Reply:
yea we have that function but that was only available for user not driver i moved it to common.

Reply:
Can you remove this log